CONFIGURATION.md

```diff
@@ -180,5 +180,3 @@ RUN_AT_EXIT_HOOKS=1 QUEUE=${QUEUE_NAME} ${EXTRA_OPTIONS} bundle exec rake resque
 Explanation:
 
 * `RUN_AT_EXIT_HOOKS`: This option, provided by Resque, ensures that the forked processes shut down gracefully (i.e., no immediate `exit!`). This allows the background processes that handle signal (trace, metrics, etc.) transmission to complete their tasks.
-
-Additionally, you need to configure the Resque initializer in your Rails application by adding the following code to `config/initializers/resque.rb`. It's recommended to have a upper bound time (e.g. 8 seconds) to avoid infinited loop if something wrong with `solarwinds_apm` initialization.
```

Comment:
@xuan-cao-swi i think this was meant to be removed as part of https://github.com/solarwinds/apm-ruby/pull/195/files#diff-749bbcf2f7f5bd5289d6223d2a4cb36ac92ff1960ca2461b01cbd732ebfbd308L192-L206

But i'm also wondering if we really don't need to talk about the need for `solarwinds_ready?`. Are you finding that the settings are pretty much always retrieved by the time the job starts executing?

Reply:
> Are you finding that the settings are pretty much always retrieved by the time the job starts executing?

It's pretty quick execution, and I think it will always retrieved by the time the job starts executing based on my observation from testing last time.